### PR TITLE
Add vaib extension

### PR DESCRIPTION
**Pull request by Vaib215**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by Vaib215** [comment]
Please approve the extension

File: extensions/vaib/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+- Run `npm run lint` to lint the code
+- Run `npm run fix-lint` to fix linting issues
+
+## Changelog
+
+See [CHANGELOG.md](CHANGELOG.md) for more information.
+
+## License
+
+This project is licensed under the MIT License.
```
Comment (pernielsentikaer) [inline extensions/vaib/README.md:34]: Please check the terms of service of third-party services that your extension uses. If your extension doesn't comply with their terms, include a warning in your extension's README. The warning should be similar to:

> Warning: This extension is not compliant with the Terms of Service of [service name]. Use at your own risk.

File: extensions/vaib/src/clipboard.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  const { push } = useNavigation();
+
+  useEffect(() => {
+    Clipboard.readText().then((text) => {
+      setQuestion(text as string);
+    });
+  }, []);
+
+  return (
+    <List navigationTitle="Search Beers" searchBarPlaceholder="Search your favorite beer">
```
Comment (pernielsentikaer) [inline extensions/vaib/src/clipboard.tsx:34]: Beers?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- Could you look into the CI errors

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/vaib/README.md (hunk 1 above)
Comment (Vaib215) [reply extensions/vaib/README.md:34]: No requirement

**Comment by Vaib215** [comment]
#10092 is the continuation

**Pull request closed without merging**